### ISSUE TITLE
[enterprise-4.9] Issue in file support/remote_health_monitoring/about-remote-health-monitoring.adoc

### ISSUE THREAD

**Issue by sdhange** [openshift/openshift-docs#40544]
### Which section(s) is the issue in?
Below sections need an update: 

1.  https://docs.openshift.com/container-platform/4.9/support/remote_health_monitoring/about-remote-health-monitoring.html#what-information-is-collected_about-remote-health-monitoring
2. https://docs.openshift.com/container-platform/4.9/support/remote_health_monitoring/about-remote-health-monitoring.html#insights-operator-what-information-is-collected_about-remote-health-monitoring

### What needs fixing?
For the first section "Information collected by Telemetry" add the below information:
 * It contains all pending/firing alerts. 
 * Also what network, infrastructure providers are being used.

For the second section "Information collected by Insights Operator" add the below information:
 * Node information (including disk/mem pressure)
 * Installed OLM operators
 * Network/netnamespaces information (possibly anonymized)
 * Information about pods that have been having issues recently
   (restarts/errors).
 * Various other cluster configuration pieces. Insights archive is a
   basically a mini-must-gather

NOTE: This was reported by @sochotnicky. @sochotnicky please give the necessary technical details as required.

**Comment by sochotnicky** [comment]
For what it's worth - when I was providing the above feedback it was *not* for the OpenShift docs. And most of the things above are already covered in the OpenShift docs, just maybe in a more generic way. For example instead of "pending/firing alerts" there is "Information about degraded software". 

I haven't gone through the list in detail but I suspect most of these things are covered in the docs in some form.
